### PR TITLE
CONCF-599 show advanced link on small resolution and move logic to correct place

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

**Comment by bognix** [comment]
@vforge @hakubo @Warkot

File: extensions/wikia/Search/css/WikiaSearch.scss [hunk 1]
```diff
@@ -639,3 +641,39 @@ $form-negative-side-padding: $form-side-padding*-1;
...
+		.search-tabs {
+			float: left;
+			margin: 35px 0 0 0;
+			width: 170px;
+		}
+
+		.results-wrapper {
+			@include width-calc("100% - 500px");
+			@media #{$breakpoint-below-small-plus} {
+				@include width-calc("100% - 180px");
```
Comment (Warkot) [inline extensions/wikia/Search/css/WikiaSearch.scss]: What are `500px` and `180px` values standing for? Something that has actually such widths? If yes, you could put them in a descriptive variable. If these are just magic numbers, give them a comment or sth.

**Comment by vforge** [comment]
LGTM

File: extensions/wikia/Search/css/WikiaSearch.scss (hunk 1 above)
Comment (bognix) [inline extensions/wikia/Search/css/WikiaSearch.scss]: I have no idea what they are. I just move this chunk of code from one place to another.

File: extensions/wikia/Search/css/WikiaSearch.scss [hunk 2]
```diff
@@ -639,3 +641,39 @@ $form-negative-side-padding: $form-side-padding*-1;
...
+			}
+
+			.grid-1.alpha {
+				width: 170px;
+				float: left;
+			}
+
+			@media #{$breakpoint-below-small-plus} {
+				.search-v2-input {
+					width: 280px;
```
Comment (bognix) [inline extensions/wikia/Search/css/WikiaSearch.scss]: **This is the only change I made**

Comment (Warkot) [inline extensions/wikia/Search/css/WikiaSearch.scss]: Oh, k. LGTM, then. :+1:

**Comment by Warkot** [comment]
BTW, is it already `tomorrow`? :smirk: https://github.com/Wikia/app/blob/CONCF-599/extensions/wikia/Search/css/WikiaSearch.scss#L688 (@RafLeszczynski authored on 4 Mar 2013)

**Comment by hakubo** [comment]
LGTM

File: extensions/wikia/Search/css/WikiaSearch.scss (hunk 1 above)
Comment (vforge) [inline extensions/wikia/Search/css/WikiaSearch.scss]: :+1:

**Pull request merged**